augraphy/augmentations/subtlenoise.py

```diff
@@ -22,17 +22,40 @@ def __init__(
     ):
         super().__init__(p=p)
         self.range = range
-        self.add_subtle_noise = np.vectorize(
```

Comment:
I removed this section and added vectorized method with numpy (line 36 below). With this, the processing time is reduced to <0.5 seconds from the previous 10 seconds ++.